lib/engine/g_1822/minor.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require_relative '../minor'
+
+module Engine
+  module G1822
+    class Minor < Minor
+      attr_accessor :type, :removed, :share_price, :max_share_price
+      attr_writer :par_price
```

Comment:
1867 has minors with share prices. Is there a way we can standardize instead of duplicating?

Reply:
Same as color, dont know much i can add to the base code.

Reply:
Code duplication is bad and should be avoided where possible.

Reply:
remove this class

Reply:
Do you mean move the share_price, par_price and max_share_price to base class?

Reply:
you don't need those things in a minor. if you have shares, then it's a corporation not a minor, thus you should remove this 1822 specific minor